runtime/Cargo.toml

```diff
@@ -46,9 +46,17 @@ sp-version = { default-features = false, git = "https://github.com/paritytech/su
 
 # Integritee dependencies
 pallet-parentchain = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "master" }
+pallet-evm = { default-features = false, optional = true, git = "https://github.com/integritee-network/frontier.git", branch = "polkadot-v0.9.24" }
 
 [features]
 default = ['std']
+# Compile the sgx-runtime with evm-pallet support in `no_std`.
+evm = ["pallet-evm"]
+# Compile the sgx-runtime with evm-pallet support in `std`.
+evm_std = [
```

Comment:
Is there an actual use case where the sgx-runtime will need to import a std compatible evm pallet?

Reply:
I can only think of `cargo test`, but I think this is worth it.

Reply:
Trusting you there :) Thanks for the cleanup, it looks good!